network-api/networkapi/wagtailpages/pagemodels/customblocks/current_events_slider_list_block.py

```diff
@@ -0,0 +1,39 @@
+from wagtail.core import blocks
+from wagtail.images.blocks import ImageChooserBlock
+
+from .common.link_blocks import LabelledExternalLinkBlock, LabelledInternalLinkBlock
+
+
+class CurrentEventBlock(blocks.StructBlock):
+    title = blocks.CharBlock('Heading for the card.')
+
+    subheading_link = blocks.StreamBlock(
+        [
+            ('internal', LabelledInternalLinkBlock()),
+            ('external', LabelledExternalLinkBlock()),
+        ],
+        max_num=1,
+    )
+
+    image = ImageChooserBlock()
+
+    body = blocks.TextBlock(help_text='Body text of the card.')
+
+    buttons = blocks.StreamBlock(
```

Comment:
Suggestion: We may need a document link block option as well here for iCal files